app/lib/query_count_job_logger.rb

```diff
@@ -2,13 +2,30 @@
 require 'sidekiq/job_logger'
 
 class QueryCountJobLogger < ::Sidekiq::JobLogger
+  SENSITIVE_KEYS = %w[
+    email
```

Comment:
I wonder if we can get this list from configuration and share it between the two logger implementations?